### DISCUSSION TITLE
enable_hyperlink vs code

### DISCUSSION THREAD

**Discussion by Mr-Vipi** [JanDeDobbeleer/oh-my-posh#1096] [Troubleshoot] [answered]
First  I don't know if oh my posh hyperlink properties for the path segments, are meant to work on vs code terminals like they do for the windows terminal. So maybe is something that just can't be implemented and fine, even though I would like to have the same functionality in both terminals.

Second all the themes that have the hyperlink property enabled in the path segment show a backslash symbol ```\``` at the begining of the first segment. This thing happens only in the git bash shell under vs code terminal. In git bash shell doesn't work only the functionality, windows terminal no issue at all. As for powershell no issue at all for the same tests.

I'm running vs code 1.61.2 and Windows 10.0.19043.0 with the lastest updates installed.
Below the issue:

![Animation1](https://user-images.githubusercontent.com/58825526/138116253-932cdbab-56bf-4270-adec-b06f15b5b193.gif)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Mr-Vipi the hyperlinks do not work in vscode as far as I know and that's a feature request for them, not us ;-)

Do know I'm not planning on supporting git bash actively, it ALWAYS has issues because of its nature.

**Reply by Mr-Vipi** [reply]
I was just wondering, but it's fine since pwsh is meant to be used in windows, it makes sense. Thanks for the prompt answer, anyway I was trying to make a custom theme, I'll be glad to share it.

**Reply by JanDeDobbeleer** [reply]
Don't get me wrong, I'm always supporting as much as I can, but Git Bash is one of the more challenging ones as it's a fake Unix env on Windows with all strange side effects etc. If I can fix it, I will :-)

**Reply by lnu** [reply]
It seems vscode terminal has its own kind of hyperlink management:

- ctrl-click on a path open a search box at the top: 
![image](https://user-images.githubusercontent.com/1829553/138215173-f29acfad-4c0a-40eb-ae93-16624e22a189.png)

- ctrl-click on a branch name opens the list of commits: 
![image](https://user-images.githubusercontent.com/1829553/138215164-30b1bf8d-edc2-4295-b94d-a3afc8bd7e92.png)
